components/Sections/NewHeader/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { FormattedMessage } from 'react-intl';
+import Image from 'next/image';
+import { MdSearch } from 'react-icons/md';
+import Link from 'next/link';
+import styles from './index.module.scss';
+import ActiveLocalizedLink from '../../ActiveLocalizedLink';
+import DarkModeToggle from '../../Common/DarkModeToggle';
+
+const Header = () => (
+  <nav aria-label="Primary" className={styles.header}>
+    <div className={styles.container}>
+      <div className={styles.startWrapper}>
+        <Link href="/" aria-label="Homepage">
+          <div className={styles.logo}>
+            <div>
+              <Image
+                src={'static/images/logo-light.svg'}
+                alt="light-logo"
+                className="light-mode-only"
+                width="111"
+                height="33"
+              />
+            </div>
+            <div>
+              <Image
+                src={'static/images/logo.svg'}
+                alt="dark-logo"
+                className="dark-mode-only"
+                width="111"
+                height="33"
+              />
+            </div>
+          </div>
+        </Link>
+      </div>
+
+      <ul className={styles.tabs}>
```

Comment:
IMHO the order should be like that:
`Learn` -> `download` -> `API docs` -> `about` -> `certification`

Why:
Learn have to be the first because this section is a good entry point for nodejs ecosystem.
Download is second because it is one of the most important pages 
API docs is next because it is explanatory of everything.
And certification because it is the " less " used

Reply:
I agree about the order. `About` & `Certification` I left out for now since it wasn't on figma.